test/vaadin-combo-box.html

```diff
@@ -65,6 +65,15 @@
         });
       });
 
+      describe('Direction', () => {
```

Comment:
If I got it correctly, this should override whenever was defined on the document level and set by `DirMixin`. Can we add a test for that? Like, if setting the document dir to RTL and `combo-box` to LTR, the overlay would have the one defined for the `combo-box`?

Reply:
Have enhanced the test with setting `documentElement` `dir`